meteomatics/tests/integration_test.rs

```diff
@@ -0,0 +1,785 @@
+use rust_connector_api::APIClient;
+use chrono::{Duration, Utc, prelude::*};
+use dotenv::dotenv;
+use std::env;
+use rust_connector_api::location::{Point, BBox};
+use polars::prelude::*;
+use std::io::Cursor;
+use std::fs;
+use std::path::Path;
+
+// Unit testing section
+// TODO: Add option to query for a grid
+// TODO: Add option to query for a grid timeseries
+#[tokio::test]
+async fn call_query_time_series_with_options() {
+    println!("\n##### call_query_time_series_with_options:");
+
+    // Credentials
+    dotenv().ok();
+    let api_key: String = env::var("METEOMATICS_PW").unwrap();
```

Comment:
Maybe instead of unwrap tell to the user, or runner of these tests that he/she has to have these env variables set, otherwise the error (panic) might be misleading.